features/support/action/scrollToElement.js

```diff
@@ -0,0 +1,23 @@
+const assert = require('assert').strict;
+
+/**
+ * Scrolls the element on which it's called into the visible area of the browser window.
+ * @param {String} selector The selector of the element to scroll to.
+ */
+module.exports = async function(selector) {
+
+    let prob = null;
+    /* istanbul ignore next */  // Required otherwise code coverage evaluation fails within $eval calls
+        try {
```

Comment:
This is 100% a sign of my OCD, but can  you fix up the indenting on this code block please :)